packages/expo-network/package.json

```diff
@@ -28,11 +28,5 @@
   },
   "author": "650 Industries, Inc.",
   "license": "MIT",
-  "homepage": "https://docs.expo.io/versions/latest/sdk/network/",
-  "devDependencies": {
-    "expo-module-scripts": "~1.2.0"
-  },
```

Comment:
This removes `devDependencies`, is this intentional?

Reply:
woops, thanks for catching